Tasks/Xcode/xcode.ts

```diff
@@ -330,14 +330,16 @@ async function run() {
                 let exportOptionsPlist: string;
                 let archiveToCheck: string = archiveFolders[0];
                 let embeddedProvProfiles: string[] = tl.findMatch(archiveToCheck, '**/embedded.mobileprovision', { followSymbolicLinks: false, followSpecifiedSymbolicLink: false });
-
+                let cloudEntitlement: boolean = false;                    
+                
                 if (exportOptions === 'auto') {
                     // Automatically try to detect the export-method to use from the provisioning profile
                     // embedded in the .xcarchive file
                     if (embeddedProvProfiles && embeddedProvProfiles.length > 0) {
                         tl.debug('embedded prov profile = ' + embeddedProvProfiles[0]);
                         exportMethod = await sign.getProvisioningProfileType(embeddedProvProfiles[0]);
                         tl.debug('Using export method = ' + exportMethod);
+                        cloudEntitlement = await sign.includesCloudEntitlement(embeddedProvProfiles[0]);
```

Comment:
It will be good if you can pass the exportMethod and the embeddedProvProfiles[0] and just get the cloud entitlement here. Change the method to getCloudEntitlement(), return Production/Development or empty string/null.

Reply:
@madhurig updated, thanks!